source/_integrations/redgtech.markdown

```diff
@@ -0,0 +1,204 @@
+---
+title: Redgtech
+description: Integrate Redgtech smart devices with Home Assistant
+ha_category:
+  - Light
+  - Switch
+ha_release: 2025.1
+ha_iot_class: cloud_polling
+ha_domain: redgtech
+ha_config_flow: true
+ha_platforms:
+  - light
+  - switch
+ha_quality_scale: bronze
+ha_codeowners:
+  - '@jonhsady'
+  - '@luan-nvg'
+---
+
+The **Redgtech** integration allows you to control your [Redgtech](https://redgtech.com.br/) smart devices through Home Assistant. This integration provides seamless connectivity to your Redgtech ecosystem, enabling you to manage lights and switches from within your Home Assistant dashboard.
+
+
+
+## Prerequisites
+
+- A Redgtech account with active devices.
+- Your Redgtech devices connected to your network.
+- Internet connectivity for cloud communication.
+
+## Installation parameters
+
+The Redgtech integration does not require any special installation parameters. The integration uses cloud-based communication and does not require:
+
+- Local network configuration
+- Port forwarding
+- Static IP addresses
+- Special firewall rules
+
+All communication is handled through the Redgtech cloud service using standard HTTPS connections.
+
+
+{% include integrations/config_flow.md %}
+
+{% configuration_basic %}
+Email:
+  description: "The email address connected to your Redgtech account."
+Password:
+  description: "The account password."
+{% endconfiguration_basic %}
+
+
+## Features
+
+- **Smart switch control**: Turn your Redgtech switches on/off remotely.
+- **Smart light control**: Control brightness and on/off state of Redgtech lights.
+- **Real-time status**: See the current state of your devices in real time.
+- **Automation support**: Use Redgtech devices in Home Assistant automations.
```

Comment:
All Home Assistant integration allow automating, it's not a special feature of this integration